src/main/java/codeu/model/store/basic/ConversationStore.java

```diff
@@ -100,4 +100,9 @@ public Conversation getConversationWithTitle(String title) {
   public void setConversations(List<Conversation> conversations) {
     this.conversations = conversations;
   }
+
+  /** Gets number of conversations in ConversationStore. */
+  public int getNumOfConversations() {
```

Comment:
I love that you're defining these new methods in Conversation/Message/UserStore - it's very "object oriented" of you :)